src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -183,7 +196,33 @@ def get(self, request, organization):
 
         results = list(cursor_result)
 
-        context = serialize(results, request.user, serializer())
+        snuba_filters = []
+        if "search_filters" in query_kwargs and query_kwargs["search_filters"] is not None:
+            snuba_filters = [
+                convert_search_filter_to_snuba_query(search_filter)
+                for search_filter in query_kwargs["search_filters"]
+                if search_filter.key.name not in self.skip_snuba_fields
+            ]
+
+        lifetime_stats = serialize(results, request.user, serializer())
+
+        if features.has("organizations:dynamic-issue-counts", organization, actor=request.user):
+            context = serialize(results, request.user, serializer(start=start, end=end))
+            if len(snuba_filters) > 0:
```

Comment:
You should be able to do `if snuba_filters` as empty lists are falsey.

Reply:
Will do